README.md

```diff
@@ -52,6 +52,12 @@ As a convenience, the following commands override some config for you:
 
 Visit `localhost:3001` to see the UI locally.
 
+### API Responses
+
+There is a sample of the API responses that we receive from Platform in `sampleApiResponseStructure.json`. It is abbreviated but shows how we receive filters and
```

Comment:
I think you probably want to be more specific saying it is the response for requesting a bib page as the app makes various requests for different kind of data.

Reply:
Thanks, I'll make some edits